model/src/main/java/mops/rollen/Dozent.java

```diff
@@ -15,9 +15,14 @@
 @EqualsAndHashCode(of = "uuid")
 @AllArgsConstructor
 public class Dozent {
-
   private UUID uuid;
   private String vorname;
   private String nachname;
   private List<Veranstaltung> veranstaltungen;
```

Comment:
Do we really still need the List with Veranstaltung as Aggregate?

Reply:
No, we don't. I'll remove it tomorrow.